installation fails on Linux (can't find portaudio.h)

`pip install rtmixer` fails on Linux 20.04. Setup:
```
$ conda create -c conda-forge -n testrtmixer python=3
$ conda activate testrtmixer
$ pip install rtmixer
```
This yields the following error:
```
    build/temp.linux-x86_64-3.8/_rtmixer.c:569:10: fatal error: portaudio.h: No such file or directory
      569 | #include <portaudio.h>
          |          ^~~~~~~~~~~~~
    compilation terminated.
```
...but the library is definitely installed:
```
$ apt search ^libportaudio2
Sorting... Done
Full Text Search... Done
libportaudio2/focal,now 19.6.0-1build1 amd64 [installed]
  Portable audio I/O - shared library
```
...and yet portaudio.h is not there:

```
$ sudo find / -name portaudio.h  # no output except a couple "permission denied" lines in `/run/usr`
```

If I install portaudio through conda:

```
$ conda install -c conda-forge portaudio  # it also installs alsa-lib
$ pip install rtmixer
```
it still yields the same error.  This time I'll paste the full output:

<details>

```
Building wheels for collected packages: rtmixer
  Building wheel for rtmixer (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /opt/miniconda3/envs/testrtmixer/bin/python3.8 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ciif4x1a/rtmixer/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ciif4x1a/rtmixer/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-419j4fk6
       cwd: /tmp/pip-install-ciif4x1a/rtmixer/
  Complete output (21 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.8
  copying src/rtmixer.py -> build/lib.linux-x86_64-3.8
  running build_ext
  generating cffi module 'build/temp.linux-x86_64-3.8/_rtmixer.c'
  creating build/temp.linux-x86_64-3.8
  building '_rtmixer' extension
  creating build/temp.linux-x86_64-3.8/build
  creating build/temp.linux-x86_64-3.8/build/temp.linux-x86_64-3.8
  creating build/temp.linux-x86_64-3.8/portaudio
  creating build/temp.linux-x86_64-3.8/portaudio/src
  creating build/temp.linux-x86_64-3.8/portaudio/src/common
  gcc -pthread -B /opt/miniconda3/envs/testrtmixer/compiler_compat -Wl,--sysroot=/ -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Isrc -Iportaudio/include -I/opt/miniconda3/envs/testrtmixer/include/python3.8 -c build/temp.linux-x86_64-3.8/_rtmixer.c -o build/temp.linux-x86_64-3.8/build/temp.linux-x86_64-3.8/_rtmixer.o --std=c99
  build/temp.linux-x86_64-3.8/_rtmixer.c:569:10: fatal error: portaudio.h: No such file or directory
    569 | #include <portaudio.h>
        |          ^~~~~~~~~~~~~
  compilation terminated.
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for rtmixer
  Running setup.py clean for rtmixer
Failed to build rtmixer
Installing collected packages: rtmixer
    Running setup.py install for rtmixer ... error
    ERROR: Command errored out with exit status 1:
     command: /opt/miniconda3/envs/testrtmixer/bin/python3.8 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ciif4x1a/rtmixer/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ciif4x1a/rtmixer/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-m7ceidgi/install-record.txt --single-version-externally-managed --compile --install-headers /opt/miniconda3/envs/testrtmixer/include/python3.8/rtmixer
         cwd: /tmp/pip-install-ciif4x1a/rtmixer/
    Complete output (21 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.8
    copying src/rtmixer.py -> build/lib.linux-x86_64-3.8
    running build_ext
    generating cffi module 'build/temp.linux-x86_64-3.8/_rtmixer.c'
    creating build/temp.linux-x86_64-3.8
    building '_rtmixer' extension
    creating build/temp.linux-x86_64-3.8/build
    creating build/temp.linux-x86_64-3.8/build/temp.linux-x86_64-3.8
    creating build/temp.linux-x86_64-3.8/portaudio
    creating build/temp.linux-x86_64-3.8/portaudio/src
    creating build/temp.linux-x86_64-3.8/portaudio/src/common
    gcc -pthread -B /opt/miniconda3/envs/testrtmixer/compiler_compat -Wl,--sysroot=/ -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Isrc -Iportaudio/include -I/opt/miniconda3/envs/testrtmixer/include/python3.8 -c build/temp.linux-x86_64-3.8/_rtmixer.c -o build/temp.linux-x86_64-3.8/build/temp.linux-x86_64-3.8/_rtmixer.o --std=c99
    build/temp.linux-x86_64-3.8/_rtmixer.c:569:10: fatal error: portaudio.h: No such file or directory
      569 | #include <portaudio.h>
          |          ^~~~~~~~~~~~~
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    ----------------------------------------
ERROR: Command errored out with exit status 1: /opt/miniconda3/envs/testrtmixer/bin/python3.8 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-ciif4x1a/rtmixer/setup.py'"'"'; __file__='"'"'/tmp/pip-install-ciif4x1a/rtmixer/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-m7ceidgi/install-record.txt --single-version-externally-managed --compile --install-headers /opt/miniconda3/envs/testrtmixer/include/python3.8/rtmixer Check the logs for full command output.
```

</details> 

What am I doing wrong here?

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -6,5 +6,6 @@ recursive-include doc *.rst *.py
 recursive-include examples *.py
 include portaudio/LICENSE.txt
 include portaudio/index.html
+include portaudio/include/portaudio.h
 include portaudio/src/common/pa_ringbuffer.h
 include portaudio/src/common/pa_memorybarrier.h

